### PR TITLE
Socket Fix

### PR DESCRIPTION
**Pull request by Rogersiver**

### CODE REVIEW

**Comment by Beauxvice** [comment]
able to merge with no problems

**Comment by calebbbbs** [comment]
no issues, glitch fixed

File: client/src/components/Nav/Nav.tsx [hunk 1]
```diff
@@ -31,25 +31,11 @@ import CreatePlaylist from './CreatePlaylist';
 
 function Nav(props: any) {
```
Comment (copelka) [inline client/src/components/Nav/Nav.tsx:32]: don't use function declarations

File: yarn.lock [hunk 2]
```diff
@@ -1850,11 +1850,20 @@
   resolved "https://registry.yarnpkg.com/@types/node/-/node-16.7.1.tgz#c6b9198178da504dfca1fd0be9b2e1002f1586f0"
   integrity sha512-ncRdc45SoYJ2H4eWU9ReDfp3vtFqDYhjOsKlFFUDEn8V1Bgr2RjYal8YT5byfadWIRluhPFU6JiDOl0H6Sl87A==
 
+<<<<<<< HEAD
```
Comment (copelka) [inline yarn.lock:1853]: fix this merge conflict

**Review by copelka — CHANGES_REQUESTED** [review]
see comments

**Pull request merged**